x-pack/test/api_integration/apis/security/privileges.ts

```diff
@@ -63,7 +63,7 @@ export default function ({ getService }: FtrProviderContext) {
             uptime: ['all', 'read'],
             apm: ['all', 'read'],
             ml: ['all', 'read'],
-            siem: ['all', 'read'],
+            siem: ['all', 'read', 'minimal_all', 'minimal_read', 'cases_all', 'cases_read'],
```

Comment:
Out of curiosity, what does this exactly does?

Reply:
This is saying that `siem` is now taking advantage of sub-feature privileges. We create the `minimal_all` and `minimal_read` privileges automatically to represent the full set of privileges for the feature, **minus** any privileges granted by sub-features. The existing `all` and `read` privileges contain all privileges granted by sub-features by default.

Reply:
I see. Thank you @legrego!